src/main/java/com/example/booksstore/models/Book.java

```diff
@@ -5,11 +5,13 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Entity
 @Data
```

Comment:
%Menthor's comment%
"this is not critical for now, but @Data is not recommended for use with @Entity, details on that:
https://medium.com/@dev.fethi/data-anatation-and-hibernate-n-1-problems-466df982288e"